src/components/structures/LeftPanel.tsx

```diff
@@ -453,8 +432,15 @@ export default class LeftPanel extends React.Component<IProps, IState> {
             "mx_AutoHideScrollbar",
         );
 
+        const panelDimensions = UIStore.instance.getElementDimensions("LeftPanel");
+
         return (
             <div className={containerClasses} ref={this.ref}>
+                <BackdropPanel
+                    backgroundImage={this.props.backgroundImage}
+                    width={panelDimensions?.width}
+                    height={panelDimensions?.height}
+                />
```

Comment:
Hmm, will having the LeftPanel and SpacePanel use separate BackdropPanels not mean your Avatar is displayed twice rather than once spanning them both?

Reply:
Trying this out, that seems to be the case, I think

Reply:
yup, the intended design is that it's a single blurred image, with the SpacePanel having less transparency (and a different blur radius) than the LeftPanel.  The intention is to model the combined panel as a hunk of frosted glass, with one side thicker than the other, hence different opacity/blur-radius.  (There could also be some scope for refraction causing an translation offset between the two, but i don't think we do that currently).

Reply:
There are still two canvases so that we're able to leverage the `filter()` function on the `Canvas2dRenderingContext` but it draws the avatar only once across the Space panel and the room list